core/iwasm/libraries/libc-wasi/libc_wasi_wrapper.c

```diff
@@ -1070,7 +1077,7 @@ wasi_poll_oneoff(wasm_exec_env_t exec_env, const wasi_subscription_t *in,
     err = wasmtime_ssp_poll_oneoff(curfds, in, out, nsubscriptions, &nevents);
 #else
     err = execute_interruptible_poll_oneoff(module_inst, curfds, in, out,
-                                            nsubscriptions, &nevents);
+                                            nsubscriptions, &nevents, exec_env);
```

Comment:
we probably don't need to pass `module_inst` if using `wasm_cluster_is_thread_terminated`